src/Monomer/Graphics/NanoVGRenderer.hs

```diff
@@ -237,6 +237,10 @@ newRenderer c rdpr envRef = Renderer {..} where
     gradient <- makeRadialGradient c dpr p1 rad1 rad2 color1 color2
     VG.strokePaint c gradient
 
+  setStrokeBoxGradient rect rad feather color1 color2 = do
```

Comment:
I added this for completeness, but it is not used inside Monomer, so could be removed.

Reply:
I thinks it's good to have it.